packages/ozone-typescript-client/src/index.ts

```diff
@@ -950,14 +701,7 @@ export namespace OzoneClient {
 		}
 	}
 
-	export interface ClientConfiguration {
-		ozoneURL: string
-		ozoneInstanceId?: string
-		ozoneCredentials?: OzoneCredentials
-		webSocketsURL?: string
-		defaultTimeout?: number
-	}
-
+	export interface ClientConfiguration extends clientConfiguration.ClientConfiguration {}
```

Comment:
Why do you have to do this? Why not simply export the original interface?

Reply:
I'don't need I should only do 
export import ClientConfiguration = clientConfiguration.ClientConfiguration